ibmpairs/paw.py

```diff
@@ -498,15 +513,20 @@ def __init__(
         else:
             self.publish2GUI    = False
         # set GUI password (default to PAIRS API password)
-        self.guiPassword        = guiPassword if guiPassword is not None else self.auth[1]
+        if type(auth) is authentication.OAuth2:
```

Comment:
`auth` may be `None` here when `self.auth` is set on Line 493

Reply:
we fix by `auth` to `self.auth` in line 516